src/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Todo App JavaScript</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w==" crossorigin="anonymous"/>
+</head>
+<body>
+  <div class="wrapper">
+    <header>Today's To Do</header>
+    <hr>
+    <div class="inputField">
+      <input type="text" id="input-box" placeholder="Add to your list ...." class="no-outline">
+    </div>
+    <hr>
+    <ul class="todoList">
+  
+    </ul>
+    <div class="footer">
+      <span class="clear-all-completed">Clear All Completed</span>
+    </div>
+  </div>
+  
+  <script type="module" src="./index.js"></script>
```

Comment:
No need to call the index.js file here in HTML as you import it in webpack.config.js

Reply:
Thank you for the review. I fixed it